docs/data/base/components/switch/UseSwitchesBasic.tsx

```diff
@@ -76,27 +87,32 @@ const BasicSwitchInput = styled('input')`
   margin: 0;
 `;
 
-const BasicSwitchThumb = styled('span')`
+const BasicSwitchThumb = styled('span')(
+  ({ theme }) => `
   display: block;
   width: 16px;
   height: 16px;
-  top: 4px;
+  top: 3px;
   left: 4px;
   border-radius: 16px;
   background-color: #fff;
+  border: 1px solid ${theme.palette.mode === 'dark' ? grey[700] : grey[300]};
   position: relative;
   transition-property: all;
   transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
   transition-duration: 120ms;
+  box-shadow: 0px 1px 2px
+    ${theme.palette.mode === 'dark' ? 'rgba(0, 0, 0, 0.25)' : 'rgba(0, 0, 0, 0.1)'};
 
   &.Switch-focusVisible {
     background-color: ${grey[500]};
     box-shadow: 0 0 1px 8px rgb(0 0 0 / 0.25);
   }
 
   &.Switch-checked {
-    left: 20px;
-    top: 4px;
+    left: 28px;
+    top: 3px;
```

Comment:
```suggestion
    left: 17px;
```

We can remove the `top` as it's declared above and adjust the `left` property.